small addition to fix Issue #175 (#180)
uptane/uptane-standard: uptane-standard.md

#### Load and verify the latest attested time {#verify_time}

> The ECU SHALL load and verify the current time, or the most recent securely attested time. This is optional if the ECU does not have the capacity to verify a time message. 

Reviewer: Can we say something SHALL be done and then immediately contradict that? Is this a CONDITIONALLY REQUIRED thing? The proposed language is fine in the abstract, but since we want to be careful with these terms, I'm just wondering if this is the best solution.

In the other instance, I don't think we have much of a choice, since the whole list is preceded by a SHALL.

Reviewer 2: Agree with Patrick here--I think we should probably relax this formally to a conditionally required. My understanding is that we cannot add any SHOULDs or MUSTs before 2.0, but I would think that relaxing a requirement would be ok.

```suggestion
IF the ECU has the capability to verify a time message, the ECU is CONDITIONALLY REQUIRED to load and verify the current time, or the most recent securely attested time.
```

Author: Hmmmm...you have a point. We can't change the SHALL since that would be a breaking change could we somewhat get around this by taking out the "optional" phrase. What is we changed it to the suggested text below. It may be equally problematic, but it somewhat addresses your concern.
```suggestion
The ECU SHALL load and verify the current time, or the most recent securely attested time if the ECU has the capacity to verify a time message. 
```

Author: This option works for me @tkfu. @iramcdonald, are we correct in changing this to a conditionally required element?

Reviewer 3: Also, shouldn't we add this language to `{{verify_time}}` instead of all the things (we could miss) that refer to it?

Author: @trishankatdatadog .. not sure what your comment refers to...I'm going to commit @tkfu suggestion above.
